Testscripts/Linux/nested_vm_utils.sh

```diff
@@ -46,6 +46,12 @@ Install_KVM_Dependencies()
         SetTestStateSkipped
         exit 0
     fi
+    lscpu | grep -i vt
```

Comment:
Can you add the comment on what it looks for?

Reply:
+1 add comments about business logic is helpful to understand intention of this change. Don't repeat what code is doing, write why it's needed, and how it works.
In this case, you can also add original input of `grep` to help understanding this change..